src/data_struct/revision_vec.rs

```diff
@@ -108,8 +108,12 @@ impl<K, T> RevisionVec<K, T> {
 
     /// Iterates through all versions of all entry in a breadth-first manner.
     #[must_use]
-    pub fn bfs(&self) -> BfsQueue<T> {
-        BfsQueue::new(self)
+    pub fn chronological_iter(&self) -> BfiQueue<T> {
```

Comment:
good idea give a sensible name to this function but it gives keys in a reverse chronological order

Reply:
I renamed it back to BFS